<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorTrill</class>
 <widget class="QWidget" name="InspectorTrill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="elementName">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Trill</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Subtype:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="trillType">
       <item>
        <property name="text">
         <string>trill line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>up prall</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>down prall</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>prall prall</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>wavy line</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="resetTrillType">
       <property name="toolTip">
        <string>reset value</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="musescore.qrc">
         <normaloff>:/data/icons-dark/resetproperty.png</normaloff>:/data/icons-dark/resetproperty.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="musescore.qrc"/>
 </resources>
 <connections/>
</ui>
